lib/ember_cli/runner.rb

```diff
@@ -32,9 +34,13 @@ def run!(command)
 
     private
 
-    def write(output, streams:)
-      streams.each do |stream|
-        stream.write(output)
+    def redirect(stream, pipes:)
+      pipes.each do |pipe|
+        Thread.new do
+          Thread.current.abort_on_exception = true
+
+          IO.copy_stream(stream, pipe)
+        end
```

Comment:
If you close the input stream, `copy_stream` will end. If you want to wait until the thread is finished, you'll have to store the thread somewhere and call `wait`.